script/setup/install-critools

```diff
@@ -32,9 +32,9 @@ git clone https://github.com/kubernetes-sigs/cri-tools.git "${TMPROOT}"/cri-tool
 pushd "${TMPROOT}"/cri-tools
 git checkout "$CRITEST_COMMIT"
 make
-make install -e BINDIR=${DESTDIR:=''}/usr/local/bin
+make install -e BINDIR=/usr/local/bin
```

Comment:
Why is `DESTDIR` no longer needed?

Reply:
it's still needed...  
at one point in the history of the crictl make install it didn't use destdir .. https://github.com/kubernetes-sigs/cri-tools/commit/a6d7d6fa1ae966cfa3e91b8635930438ee22bcb9#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R71   

then it did..  but we were still concatenating and with the -e option we added at some point we started passing over our destdir and binder which already had destdir... :-)

Reply:
https://github.com/kubernetes-sigs/cri-tools/commit/a2e4fdc480d62f299295a1897a91546b174de81e#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52
^

Reply:
Could it be passed explicitly `-e DESTDIR=${DESTDIR:=''}` to show it is there and used?